usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/port/ExecParamRepository.java

```diff
@@ -35,6 +35,8 @@
     String URL_KEY = "url";
     String ZIP_KEY = "zipinput";
     String EXCLUSION_KEY = "exclude";
+    String LOWER_BOUND_MIN_TRANSFER_TIME = "mintransfertime";
```

Comment:
Try maintaining the same nomenclature throughout the code to ease legibility, ie have the same names on both sides of the equal sign, excluding camel case differences